commcare_connect/templates/tailwind/components/cards/card.html

```diff
@@ -0,0 +1,19 @@
+<!-- card.html -->
+<div class="bg-white p-4 rounded-lg shadow-lg" 
+x-data="{ message: '' }" 
+@button-clicked="message = $event.detail.message; console.log('Button clicked')">
```

Comment:
You can use alpinejs x-on dir for click binding https://alpinejs.dev/directives/on